plugins/py3embed/CMakeLists.txt

```diff
@@ -53,6 +57,44 @@ elseif(NOT APPLE)
     set_target_properties(py3embed PROPERTIES NO_SONAME 1)
 endif()
 
+if (CREATE_STDLIB_ZIP)
+    set(_py3_zip_temp_dir "${CMAKE_CURRENT_BINARY_DIR}/python_embeddable_zip_temp")
+    set(_py3_zip_path "${_py3_zip_temp_dir}/python${Python3_VERSION_MAJOR}${Python3_VERSION_MINOR}.zip")
```

Comment:
Is this name correct for debug versions?  (If so it is only a problem for developers, so could be a separate PR)

Reply:
I think this is still valid - I will recheck now...

Reply:
It was still valid for the "Lib" folder, but not for the zip option.  Fixed...